spec/system/feedback_spec.rb

```diff
@@ -29,6 +29,7 @@
         find("#comment-field")
         fill_in "comment-field", with: "some feedback!"
         stored_data = SatisfactionFeedback.find_by(satisfied: "yes", outcome: "eligible", level_of_help: "controlled")
+        sleep 1
```

Comment:
is this a general flakey test fix or to fix an issue now we are on ruby 3.3.1?

Reply:
This was an existing flake - it could be a separate PR, but it was starting to annoy me